# socket.error

### socket.error _exception_

Устаревший псевдоним OSError.

_Изменено в версии 3.3:_ после [PEP 3151](https://www.python.org/dev/peps/pep-3151/) этот класс стал псевдонимом OSError.

